pybikes/data/samba.json

```diff
@@ -140,6 +140,23 @@
                     }
                 }
             ]
+        },
+        "SambaArgentina": {
+            "instances": [
+                {
+                    "url":"http://www.mibicitubici.gob.ar/estacione.aspx",
+                    "tag":"mibicitubici",
+                    "meta": {
+                        "city": "Rosario",
+                        "name": "Mi bici tu bici",
+                        "country": "AR",
+                        "longitude": -60.65,
+                        "latitude": -32.95,
+                        "system": "Sistema de Bicicletas Públicas mi bici – tu bici",
```

Comment:
No need to declare system. It really is just an identifier of which type of system it is using. Like, on this case, it's using Samba (already declared on BaseSystem), and it will already show up.